src/kouta_indeksoija_service/indexer/cache/eperuste.clj

```diff
@@ -58,20 +62,33 @@
              :tunniste (:tunniste osaamisala)}
             (get-opintojen-laajuus (get-in muodostumissaanto [:laajuus :minimi])))))))
 
+(defn- get-diplomi-sisallot-tavoitteet [eperuste]
+  (let [oppiaineet (get-in eperuste [:lops2019 :oppiaineet])]
+    (as-> oppiaineet o
+      (filter (fn [oppiaine] (= (:id oppiaine) lukiodiplomit-oppiaine-id)) o)
+      (first o)
+      (get o :moduulit)
+      (map (fn [moduuli] [(get-in moduuli [:koodi :uri]) {:sisallot (get-in moduuli [:sisallot 0 :sisallot])
+                                                          :tavoitteet (get-in moduuli [:tavoitteet :tavoitteet])}]) o)
+      (cond (empty? o) nil :else (into {} o)))))
```

Comment:
Tää as-> ei oo tainnu olla käytössä, vaan on ollu suoraa vaan -> tai ->> jolloin ei toistu tuo o joka välissä. Ehkä tän voi jättää näin koska tää on sinällään selkeempi lukea.

Reply:
Käytin tuota `as->` jo jossain aiemminkin. Tässä vois käyttää `->>`, jos tuosta `(get o :moduulit)`-rivistä pääsis eroon tai sais `get`:in argumentit flipattua päinvastaseen järjestykseen. En vaan osaa 😄.